src/main/java/uk/gov/hmcts/ccd/CoreCaseDataConfiguration.java

```diff
@@ -2,16 +2,24 @@
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
 public class CoreCaseDataConfiguration {
 
     @Bean
+    @Qualifier("caseDetailsObjectMapper")
     public ObjectMapper objectMapper() {
```

Comment:
can you rename the method as well please

Reply:
renamed